cleanup_utils/playbooks/load_balancer_cleanup.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Playbook to remove the stale load balancer fragments
+
+- hosts: all
+  become: True
+  vars:
+    DAYS_OLDER_THAN: 3
+    FRAGMENT_DIR: /etc/haproxy/conf.d
+    FRAGMENT_PATTERN: "integration*"
+    REMOVE_FRAGMENTS: false
+  tasks:
+    - name: Find load balancer fragment files older than 'DAYS_OLDER_THAN'
+      find:
+        paths: "{{ FRAGMENT_DIR }}"
+        age: "{{ DAYS_OLDER_THAN }}d"
```

Comment:
We can't use this tactic because this would include some legit fragments, like Ocim stage. It currently deleted Ocim stage's backend fragment in stage HAProxy. We need to base it off of some regex, which I think is what you meant to do with `FRAGMENT_PATTERN`, but didn't include it here.